src/main.rs

```diff
@@ -204,8 +207,14 @@ fn run() -> Result<()> {
                                  .value_name("COMMIT-MESSAGE")
                                  .help("Commit message that will be used on import")
                                  .default_value("cobalt site import")
-                                 .takes_value(true)))
-        .get_matches();
+                                 .takes_value(true)));
+
+    #[cfg(all(feature="syntax-highlight"))]
+    let app_cli =
+        app_cli
+            .subcommand(SubCommand::with_name("list-syntax-themes").about("list available themes"));
```

Comment:
I think my reasoning at the time was without `features=syntax-highlight`, then:

1.  `list-syntax-themes` is an invalid subcommand,
2.  don't not show `list-syntax-themes` when `cobalt help` runs,
3.  bail in the subcommand match to mimic invoking the default match arm when the user invoked `list-syntax-themes` as an invalid subcommand.

Though (3) doesn't bail at the moment since *clap* generates an error for invalid subcommands.

Sounds like you don't mind `list-syntax-themes` being an available subcommand regardless whether the build includes *syntect*, so I can drop the feature toggle at [212](https://github.com/cobalt-org/cobalt.rs/pull/259#discussion-diff-126850433R212), and instead of mimicking the default subcommand behavior, bail with a message on stderr like "No themes available.  This build of cobalt does not include support for syntax highlighting".